composer.json

```diff
@@ -4,25 +4,48 @@
     "type": "library",
     "license": "MIT",
     "homepage": "https://github.com/acmephp/core",
-    "keywords": [ "ACME", "letsencrypt", "https", "encryption" ],
+    "keywords": [
+        "acmephp",
+        "letsencrypt",
+        "https",
+        "encryption",
+        "certificate",
+        "ssl",
+        "openssl",
+        "CSR",
+        "x509"
+    ],
     "authors": [
         {
             "name": "Titouan Galopin",
             "email": "galopintitouan@gmail.com",
             "homepage": "http://titouangalopin.com"
+        },
+        {
+            "name": "Jérémy Derussé",
+            "homepage": "https://twitter.com/jderusse"
+        }
+    ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:acmephp/ssl.git"
         }
     ],
     "require": {
         "php": ">=5.5.0",
         "ext-openssl": "*",
+        "acmephp/ssl": "dev-master",
         "psr/log": "^1.0",
-        "webmozart/assert": "^1.0",
-        "guzzlehttp/guzzle": "^6.1"
+        "guzzlehttp/guzzle": "^6.2"
     },
     "require-dev": {
         "phpunit/phpunit": "^4.8.22",
-        "sllh/php-cs-fixer-styleci-bridge": "^1.5.0"
+        "sllh/php-cs-fixer-styleci-bridge": "^1.5.0",
+        "symfony/process": "^3.0"
     },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
```

Comment:
copy of https://github.com/swarrot/SwarrotBundle/pull/75#discussion-diff-45033640

> it would be better to change the minimum stability only in builds needing it. Otherwise, there is no build ensuring that dependencies can be resolved using only stable releases (which is what users of the bundle need by default)